src/main/java/com/auth0/jwk/Jwk.java

```diff
@@ -55,21 +56,49 @@ public Jwk(String id, String type, String algorithm, String usage, String operat
         this.additionalAttributes = additionalAttributes;
     }
 
+    /**
+     * <p>Deprecated: Specification states that 'key_ops' (operations) is expected to contain an array value.
```

Comment:
Keep the constructor javadoc equal to the other one. But move this deprecation details below to a `@deprecated` tag in the javadoc, as explained [here](https://stackoverflow.com/a/20674323). I'd simplify the details like this:

```
@deprecated The specification states that the 'key_ops' (operations) parameter contains an array value. Use {@link #Jwk(String, String, String, String, List, String, List, String, Map)} instead.
```